backend/pkg/api/routes.go

```diff
@@ -30,6 +33,61 @@ import (
 	"github.com/redpanda-data/console/backend/pkg/version"
 )
 
+// Setup connect and grpc-gateway
+func (api *API) setupConnectGRPCGateway(r chi.Router) {
+	// Connect RPC
+	userSvc := apiusersvc.NewService(api.Cfg, api.Logger.Named("user_service"), api.RedpandaSvc, api.ConsoleSvc)
+
+	// With server reflection enabled, ad-hoc debugging tools can call your gRPC-compatible
+	// handlers and print the responses without a copy of the proto schema.
+	reflector := grpcreflect.NewStaticReflector(consolev1alphaconnect.ConsoleServiceName, dataplanev1alpha1connect.UserServiceName)
+
+	// Setup Interceptors
+	v, err := protovalidate.New()
+	if err != nil {
+		api.Logger.Fatal("failed to create proto validator", zap.Error(err))
+	}
+	// we want the actual request validation after all authorization and permission checks
+	interceptors := []connect.Interceptor{
+		interceptor.NewRequestValidationInterceptor(v, api.Logger.Named("validator")),
+	}
+
+	// Setup gRPC-GW
+	mux := runtime.NewServeMux(
+		runtime.WithErrorHandler(NiceHTTPErrorHandler),
+		runtime.WithMarshalerOption(runtime.MIMEWildcard, &runtime.HTTPBodyMarshaler{
+			Marshaler: &runtime.JSONPb{
+				MarshalOptions: protojson.MarshalOptions{
+					UseProtoNames:   true, // use snake_case
+					EmitUnpopulated: true, // output zero values e.g. 0, "", false
+				},
+				UnmarshalOptions: protojson.UnmarshalOptions{
+					DiscardUnknown: true,
+				},
+			},
+		}),
+	)
+
+	// Mount Connect services
+	r.Mount(grpcreflect.NewHandlerV1(reflector))
+	r.Mount(grpcreflect.NewHandlerV1Alpha(reflector))
+	r.Mount(consolev1alphaconnect.NewConsoleServiceHandler(consolev1alphaconnect.UnimplementedConsoleServiceHandler{}, connect.WithInterceptors(interceptors...)))
+
+	userSvcPath, userSvcHandler := dataplanev1alpha1connect.NewUserServiceHandler(userSvc, connect.WithInterceptors(interceptors...))
+	r.Mount(userSvcPath, userSvcHandler)
+	// Connect services are hooked via the ordinary
+	// api.Hooks.Route.ConfigAPIRouter() hook.
+
+	// gRPC-Gateway handlers
+	dataplanev1alpha1connect.RegisterUserServiceHandlerGatewayServer(mux, userSvc, connectgateway.WithInterceptors(interceptors...))
+	// Call gRPC-Gateway hooks
+	api.Hooks.Route.ConfigGRPCGateway(mux)
+
+	// Mount gRPC-Gateway mux. We support /api/v1 and /api/v1alpha1.
+	r.Mount("/api/v1", mux)
+	r.Mount("/api/v1alpha1", mux)
```

Comment:
The `/api` prefix to be removed here, so that we expose under `/v1` / `/v1alpha1`?